File: abc240/a/a.jl
function read_params(T::Type)
    line = readline()
    return map(x -> parse(T, x), split(line))
end

function solve()
    a, b = read_params(Int)
    if abs(a - b) == 1 || abs(a - b) == 9
        println("Yes")
    else
        println("No")
    end
end

solve()

File: abc240/a/a.dart
import 'dart:io';

void main() {
  var params = read_ints();
  var delta = (params[0] - params[1]).abs();
  if (delta == 1 || delta == 9) {
    print("Yes");
  } else {
    print("No");
  }
}

List<int> read_ints() {
  var inputText = stdin.readLineSync();
  if (inputText != null) {
    return inputText.trim().split(' ').map((e) => int.parse(e)).toList();
  } else {
    return [];
  }
}
